fuzz.go

```diff
@@ -478,7 +478,7 @@ type charRange struct {
 // choose returns a random unicode character from the given range, using the
 // given randomness source.
 func (r *charRange) choose(rand *rand.Rand) rune {
```

Comment:
I know it's unrelated, but can you change this to `r *rand.Rand`? Hiding the package name with the variable name is confusing and gave me a minor panic attack just now because this code should not be making calls to the built in rand package loose functions.

Reply:
done